docs/reference/troubleshooting/common-issues/rejected-requests.asciidoc

```diff
@@ -23,9 +23,50 @@ To check the number of rejected tasks for each thread pool, use the
 
 [source,console]
 ----
-GET /_cat/thread_pool?v=true&h=id,name,active,rejected,completed
+GET /_cat/thread_pool?v=true&h=id,name,queue,active,rejected,completed
 ----
 
+The `write` thread pool rejections frequently surface in the erring API and
+correlating log as `EsRejectedExecutionException` with either
+`QueueResizingEsThreadPoolExecutor` or `queue capacity`.
+
+This frequently relates to <<task-queue-backlog,backlogged tasks>>
```

Comment:
what is an "erring API"? if you mean the API that's erroring out, I'm not sure  where that would come into it 

when you say "with", do you mean it's a single error string with both of those substrings? 

```suggestion
`write` thread pool rejections frequently appear in the erring API and
correlating log as `EsRejectedExecutionException` with either
`QueueResizingEsThreadPoolExecutor` or `queue capacity`.

These errors are often related to <<task-queue-backlog,backlogged tasks>>.
```